app/src/main/res/layout/item_location.xml

```diff
@@ -1,42 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/ll_locations"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:layout_marginBottom="@dimen/card_margin_vertical"
-    android:layout_marginTop="@dimen/card_margin_vertical"
-    android:orientation="vertical">
+    android:orientation="vertical"
+    android:background="?android:attr/selectableItemBackground">
 
-    <android.support.v7.widget.CardView
-        android:id="@+id/card_locations"
+    <LinearLayout
+        android:orientation="vertical"
```

Comment:
This `LinearLayout` acts as a wrapper for the two `TextView` stacked one after the other. That's why the vertical `orientation`.